### PR TITLE
aesthetic updates to code.org/learn

### PR DESCRIPTION
**Pull request by tanyaparker**
This PR entails:
- [x] activate sliding animation *only for EN, not yet for non-EN
- [x] fix banner animation twitch
- [x] on mobile, move arrows to side instead of below
- [x] in i18n short links need to be overflow: hidden & ellipsis (see german)
- [x] drop line height between desc & links
- [x] make teacher lesson plans it’s own carousel
- [x] hide bee banner on i18n
- [x] add 2 links to bottom of /learn that appears for EN only -> leads to all tutorials & tutorial guidelines

Todo after code review:
- [x] sync new pegasus string to all locales

**320px width**
image

**new carousel section for teacher led lesson plans**
image

### CODE REVIEW

**Comment by tanyaparker** [comment]
@breville @caleybrock could you take a look? 

I put sliding animation for EN only, and I've tested it works across Chrome, FF, Safari, IE10+, etc. It's still being fussy for non-EN, so I've continued to keep it disabled for non-EN.

File: pegasus/sites.v3/code.org/public/css/learn-carousel.css [hunk 1]
```diff
@@ -200,16 +175,36 @@
...
+    height: 80px;
   }
-  .multi-item-carousel .carousel-inner .active.right {
-    left: 0%;
+
+  @media all and (transform-3d), (-webkit-transform-3d) {
+    .carousel-inner > .item.next,
+    .carousel-inner > .item.active.right {
+      left: 0;
+      -webkit-transform: translate3d(50%, 0, 0);
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/css/learn-carousel.css:186]: is it okay that translate3d not available in ie8/9 and only partially in ie10/11?

File: pegasus/sites.v3/code.org/public/css/common-international.css [hunk 2]
```diff
@@ -252,35 +244,43 @@
   }
 }
 
-@media screen and (min-width: 600px) /* and (max-width: 969px) */ {
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/css/common-international.css:255]: i think we should probably start adding detailed comments to our css files at some point.. carousels especially are complex!  (or maybe we should just switch to react :)

**Comment by breville** [comment]
Is teacher-led retrieved as a jpg?  we can do that and pegasus will convert.  also, is that image as big as our other carousel images?

lgtm, but a change like this is really all about the testing, so if you're confident it's broadly tested, then all good.  happy to help testing if you want to walk through anything.

**Comment by tanyaparker** [comment]
What do you mean retrieved as a jpg? Should I convert it to a jpg? It's as big as other carousel images, yep.

**Comment by breville** [comment]
You can leave it as a png in the source tree, but i believe that if you request it as the .jpg version of the same filename, pegasus will convert it.  that'll save some bandwidth.

**Comment by breville** [comment]
Also the raw image seems small.. we seem to have larger source images for carousel, e.g. https://code.org/images/scratch.jpg chosen at random

File: pegasus/sites.v3/code.org/public/css/learn-carousel.css (hunk 1 above)
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/css/learn-carousel.css:186]: Ugh, right. So no animation in non-EN and IE9- but it's no worse than it is now. Just advances by 1, no slide animation. I'll keep tracking to get full support hopefully by next week. But at least getting this fix in for Thursday deadline.

**Comment by tanyaparker** [comment]
They are all different sizes, but one of the carousels I checked had this size. I'll just change to a bigger one.

**Pull request merged**